src/core/modules/StarMgr.hpp

```diff
@@ -436,7 +437,10 @@ private slots:
 
 	//! List of all Hipparcos stars.
 	QList<StelObjectP> hipparcosStars;
-	QList<QMap<StelObjectP, float>> doubleHipStars, variableHipStars, hipStarsHighPM;
+	QList<QMap<StelObjectP, float>> doubleHipStars;
```

Comment:
This part of code may be written more compact

Reply:
ok...